fleet_vehicle_fuel_capacity/models/fleet_vehicle.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2021 - TODAY, Marcel Savegnago
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class FleetVehicle(models.Model):
+
+    _inherit = 'fleet.vehicle'
+
+    fuel_capacity = fields.Float(
+        string='Fuel Capacity (L)',
```

Comment:
Something like this maybe :
https://github.com/odoo/odoo/blob/e8ef12718ad82d9868cb245e9a95f23ba826d989/addons/product/models/product_template.py#L273 ?

Reply:
Hello Guys. As I understand it, the fleet module works like liters on vehicle fuel log. In the case of capacity, one option is to leave the fuel_capacity field without mentioning the unit of measurement and adding a new selection field for the unit of measurement (liter or gallon).

Reply:
If applicable, I can implement it in the fleet settings to define the unit of measurement to record the standard unit of measurement for a vehicle's fuel capacity.

Reply:
Not a serious problem for me at this time.  Just thought I would bring it up. 

I agree in the roadmap it would be good to add an option to choose between Liters or Gallons since core Odoo seems to have not considered those of us that are using gallons. That being said, I appreciate your recent motive on the fleet modules to fill some of these gaps!

Reply:
I think it is very valid to have brought this up. For this module, can we implement only the fuel_capacity field and think about an additional module to allow the refueling record to also work in gallons?

Reply:
Look at this. :(

Do you have any suggestions ?

![image](https://user-images.githubusercontent.com/3595132/113649946-bce63980-9665-11eb-9aa6-955766f5c2c2.png)

https://github.com/odoo/odoo/blob/b2afcb56cc42b89bad2a90e3d46b9787a0804264/addons/fleet/models/fleet_vehicle_cost.py#L310

Reply:
https://github.com/OCA/fleet/issues/60

Reply:
> I think it is very valid to have brought this up. Can we just implement the fuel_capacity field and think of an additional module to allow the refueling log to also work in gallons?

I also think that this feature can be in another module :+1: